app/src/main/java/com/example/hris/model/ResponseModel.kt

```diff
@@ -1,6 +1,6 @@
 package com.example.hris.model
 
-data class ResponseModel(
-    val status: String,
-    val message: String
-)
\ No newline at end of file
+abstract class ResponseModel {
```

Comment:
Making this class `abstract` will not allow you to use this class as-is. So if you're API only has `status` and `message` as returns, you will still need to make a new class and extend this class. E.g. `AddTimeLogsResponseModel`. Instead, just make this class `open` so you can use this as-is and still can be extended by other classes.